content/app/design/guidelines/components/checkbox/_index.en.md

```diff
@@ -1,16 +1,14 @@
 ---
-title: Avkrysningsboks
-description: Avkrysningsbokser brukes i tilfeller der brukeren kan huke av ett eller flere alternativer fra en liste. 
+title: Check box
```

Comment:
This component is named `Checkboxes` in code, so the heading should at least be consistent with that.
```suggestion
title: Checkboxes
```